code/src/main/kotlin/com/expediagroup/sdk/v2/core/client/ApiClientApolloHttpEngine.kt

```diff
@@ -0,0 +1,64 @@
+package com.expediagroup.sdk.v2.core.client
+
+import com.apollographql.apollo.api.http.HttpRequest
+import com.apollographql.apollo.api.http.HttpResponse
+import com.apollographql.apollo.exception.ApolloNetworkException
+import com.apollographql.java.client.ApolloDisposable
+import com.apollographql.java.client.network.http.HttpCallback
+import com.apollographql.java.client.network.http.HttpEngine
+import com.expediagroup.sdk.v2.core.request.Request
+import com.expediagroup.sdk.v2.core.client.util.toApolloHeaders
+import okio.buffer
+import okio.source
+
+
+/**
+ * Implementation of the `HttpEngine` interface that uses an `ApiClient` instance to execute HTTP requests.
```

Comment:
The docs is not clear, this class is mainly to adapt Apache to be used with Apollo

Reply:
Fixed.